tests/python-gpu/test_gpu_basic_models.py

```diff
@@ -55,9 +55,6 @@ def test_deterministic_gpu_hist(self):
         model_0, model_1 = self.run_cls(X, y, True)
         assert model_0 == model_1
 
-        model_0, model_1 = self.run_cls(X, y, False)
```

Comment:
Is this related to the pull request?

Reply:
Yes, now the deterministic parameter is removed, this test is no longer useful.